File: desafios/d001/mensagem.py
def msg(tam, passo=1):
    if (passo >= 1):
        for e in range(0, tam, passo):
            print(f"{'*' * (e+1)} {chr(48+e+1)}")

    if (passo < 0):
        for i in range(tam, 0, passo):
            print(f"{'*' * (i)} {chr(48+i)}")

def main():
    tam = int(input())
    msg(tam, 1)
    print('')
    msg(tam, -1)

if __name__ == "__main__":
    main()

File: desafios/d001/mensagem.js
function msg(tam, passo=1) {
    const numerais = ['Um', 'Dois', 'Tres', 'Quatro']

    if (passo >= 1) {
        for (var e = 0; e < tam; e++) {
            console.log("*" * (e+1), numerais[e]);
        }
    }
    
    // if (passo < 0) {
    //     for (var i = tam; i > 0; i--) {
    //         console.log("*" * (i+1), numerais[e-1])
    //     }
    // }
}

function main() {
    msg(4, 1);
    console.log("\n");
    // msg(4, -1);
}